Ray.jl/src/runtime.jl

```diff
@@ -159,6 +161,9 @@ function start_worker(args=ARGS)
 
     parsed_args = parse_args(args, s)
 
+    _init_global_function_manager(parsed_args["address"])
+
+    ENV["JULIA_DEBUG"] = "Ray"
```

Comment:
should this go at the top level of the module?

Reply:
probably shouldn't even be here at all (shoudl have a switch or something to turn on debug logging that gets passed in the runtime env) but was the most direct way to do it on the workers